app/templates/partials/forms/select.html

```diff
@@ -1,12 +1,17 @@
 <select class='input input--select' id='input-{{select.name}}' name='{{select.name}}'>
+
+  {% if not (select.selected and select.mandatory) %}
+      <option value="">{{select.placeholder}}</option>
+  {% endif %}
+
```

Comment:
I think we should be adding the placeholder text to the select.options in `date_widget.py`. If we need to do any logic about not displaying the placeholder text that can be done in the widget.